docs/getting-started-guides/vsphere.md

```diff
@@ -40,201 +40,114 @@ guide](/docs/concepts/storage/volumes/#vspherevolume)
+| manage-k8s-volumes | Datastore.AllocateSpace<br/>Datastore.FileManagement<br/>System.Anonymous<br/>System.Read<br/>System.View<br/> | Datastore  |  No |
+| k8s-system-read-and-spbm-profile-view |StorageProfile.View<br/>System.Anonymous<br/>System.Read<br/>System.View<br/> | vCenter  | No |
+| ReadOnly | System.Anonymous<br/>System.Read<br/>System.View<br/>  | Datacenter,<br/>Datastore Cluster,<br/>Datastore Storage Folder</br>  | No  |
 
-* Get or build a [binary release](/docs/getting-started-guides/binary_release)
 
-#### Setup
+**[ Step 5 ]** Create the vSphere cloud config file. Cloud config template can be found [here](https://github.com/kubernetes/kubernetes-anywhere/blob/master/phase1/vsphere/vsphere.conf)
 
-Download a prebuilt Debian 8.2 VMDK that we'll use as a base image:
+This config file needs to be placed in the shared directory which should be accessible from kubelet container, controller-manager pod, and API server pod.
 
-```shell
-curl --remote-name-all https://storage.googleapis.com/govmomi/vmdk/2016-01-08/kube.vmdk.gz{,.md5}
-md5sum -c kube.vmdk.gz.md5
-gzip -d kube.vmdk.gz
-```
-
-Configure the environment for govc
-
-```shell
-export GOVC_URL='hostname' # hostname of the vc
-export GOVC_USERNAME='username' # username for logging into the vsphere.
-export GOVC_PASSWORD='password' # password for the above username
-export GOVC_NETWORK='Network Name' # Name of the network the vms should join. Many times it could be "VM Network"
-export GOVC_INSECURE=1 # If the host above uses a self-signed cert
-export GOVC_DATASTORE='target datastore'
-# To get resource pool via govc: govc ls -l 'host/*' | grep ResourcePool | awk '{print $1}' | xargs -n1 -t govc pool.info
-export GOVC_RESOURCE_POOL='resource pool or cluster with access to datastore'
-export GOVC_GUEST_LOGIN='kube:kube' # Used for logging into kube.vmdk during deployment.
-export GOVC_PORT=443 # The port to be used by vSphere cloud provider plugin
-# To get datacente via govc: govc datacenter.info
-export GOVC_DATACENTER='ha-datacenter' # The datacenter to be used by vSphere cloud provider plugin
-```
-
-Sample environment
-
-```shell
-export GOVC_URL='10.161.236.217'
-export GOVC_USERNAME='administrator'
-export GOVC_PASSWORD='MyPassword1'
-export GOVC_NETWORK='VM Network'
-export GOVC_INSECURE=1
-export GOVC_DATASTORE='datastore1'
-export GOVC_RESOURCE_POOL='/Datacenter/host/10.20.104.24/Resources'
-export GOVC_GUEST_LOGIN='kube:kube'
-export GOVC_PORT='443'
-export GOVC_DATACENTER='Datacenter'
-```
-
-Import this VMDK into your vSphere datastore:
-
-```shell
-govc import.vmdk kube.vmdk ./kube/
-```
-
-Verify that the VMDK was correctly uploaded and expanded to ~3GiB:
-
-```shell
-govc datastore.ls ./kube/
+Sample Config:
```

Comment:
For worker nodes, this config could be different.
If worker nodes set vm-name in the config file, they can exclude user/password in their config files.